### PR TITLE
feat: add web3js templates

### PR DESCRIPTION
**Pull request by avkos**
# What :computer: 
This PR adds web3js templates to the CLI

[CLI PR](https://github.com/matter-labs/zksync-cli/pull/121)
[Templates PR](https://github.com/matter-labs/zksync-scripting-templates/pull/4)
[web3-zksync-plugin](https://github.com/web3/web3-plugin-zksync)

# Why :hand:
* Reason why first thing was added to PR
* Reason why second thing was added to PR
* Reason why third thing was added to PR

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

# Notes :memo:
- This PR has to be merged [AFTER the template PR](https://github.com/matter-labs/zksync-scripting-templates/pull/4)

### CODE REVIEW

File: src/commands/create/groups/scripting.ts [hunk 1]
```diff
@@ -8,10 +8,18 @@ import type { GenericTemplate } from "../index.js";
...
 type Template = GenericTemplate & {
   framework: "Node.js";
-  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem";
+  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem" | "web3js";
 };
 
 export const templates: Template[] = [
+  {
+    name: "Web3js - Node.js",
+    value: "web3js",
```
Comment (uF4No) [inline src/commands/create/groups/scripting.ts:17]:
```suggestion
    value: "node_web3js",
```
just to follow the same pattern as other templates

Comment (uF4No) [inline src/commands/create/groups/scripting.ts:17]:
```suggestion
    value: "node_web3js",
```
to follow the same pattern as the rest of the templates

File: src/commands/create/groups/scripting.ts [hunk 2]
```diff
@@ -8,10 +8,18 @@ import type { GenericTemplate } from "../index.js";
 
 type Template = GenericTemplate & {
   framework: "Node.js";
-  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem";
+  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem" | "web3js";
```
Comment (JackHamer09) [inline src/commands/create/groups/scripting.ts:11]:
```suggestion
  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem" | "Web3.js";
```

File: src/commands/create/groups/scripting.ts [hunk 3]
```diff
@@ -8,10 +8,18 @@ import type { GenericTemplate } from "../index.js";
...
 };
 
 export const templates: Template[] = [
+  {
+    name: "Web3js - Node.js",
+    value: "node_web3js",
+    framework: "Node.js",
+    ethereumFramework: "web3js",
```
Comment (JackHamer09) [inline src/commands/create/groups/scripting.ts:19]:
```suggestion
    ethereumFramework: "Web3.js",
```

**Review by JackHamer09 — CHANGES_REQUESTED** [review]

**Review by JackHamer09 — APPROVED** [review]

**Pull request merged**